docs/getting-started/dev-portal/dev-portal-options.md

```diff
@@ -1,15 +1,15 @@
 # Dev Portal Configuration Options
 
 - `projectId` - Find the Project ID in the Project Settings screen on Stoplight.
-- `platformUrl` - By default this is `https://stoplight.io/` and this default will be fine for most people. Some enterprise customers might need to change it.
+- `platformUrl` - By default, the URL is `https://stoplight.io/', but you can change it.
 - `basePath` - Helps when using `router: 'history'` but docs are in a subdirectory like `https://example.com/docs/api`.
-- `collapseTableOfContents` - Pass `true` to stop the table of contents expanding folders of articles and showing lists of endpoints for an API. This will clean up the ToC if you have a lot of content.
+- `collapseTableOfContents` - Pass `true` to initially collapse ToC folders of articles and showing lists of endpoints for an API. This can help organize the ToC if you have a lot of content.
```

Comment:
I think 'showing' is off because of the other changes; 'folders of articles and show lists of endpoints ...'

Reply:
You are right. Everything is collapsed when this is set to true, so I changed to:

Pass `true` to initially collapse ToC folders of articles and lists of endpoints for an API.